bazel_tools/haskell-cabal-collisions.patch

```diff
@@ -0,0 +1,290 @@
+From 17a2c0e7813236fdcc8107544922d44fcabe924c Mon Sep 17 00:00:00 2001
```

Comment:
I guess we can upstream this in the future?

Reply:
Yes, there's a PR for it already, it's linked in `deps.bzl` https://github.com/digital-asset/daml/pull/2743/files#diff-0d048ba7b51a63feefd2338106bd6d99R53